CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,143 @@
+
+No matter how you identify yourself or how others perceive you: we welcome you.
+Though no list can hope to be comprehensive, we explicitly honor diversity in:
+age, culture, ethnicity, genotype, gender identity or expression, language,
+national origin, neurotype, phenotype, political beliefs, profession, race,
+religion, sexual orientation, socioeconomic status, subculture and technical
+ability, to the extent that these do not conflict with this code of conduct.
+
+Though we welcome people fluent in all languages, pygfx development is
+conducted in English.
+
+Standards for behavior in the pygfx community are detailed in the Code of
+Conduct above. Participants in our community should uphold these standards
+in all their interactions and help others to do so as well (see next section).
+
+
+# Reporting guidelines
+
+We know that it is painfully common for internet communication to start at or
+devolve into obvious and flagrant abuse. We also recognize that sometimes
+people may have a bad day, or be unaware of some of the guidelines in this Code
+of Conduct. Please keep this in mind when deciding on how to respond to a
+breach of this Code.
+
+For clearly intentional breaches, report those to the Code of Conduct committee
+(see below). For possibly unintentional breaches, you may reply to the person
+and point out this Code of Conduct (either in public or in private, whatever is
+most appropriate). If you would prefer not to do that, please feel free to
+report to the Code of Conduct committee directly, or ask the committee for
+advice, in confidence.
+
+You can report issues to the pygfx core team: 
+
+[Korijn van Golen](https://github.com/Korijn)
+[Almar Klein](https://github.com/almarklein)
+
+If your report involves any members of the pygfx core team, or if they feel they have
+a conflict of interest in handling it, then they will recuse themselves from
+considering your report.
+
+# Incident reporting resolution & Code of Conduct enforcement
+
+We will investigate and respond to all complaints. The pygfx Code of Conduct
+Committee and the pygfx Steering Committee (if involved) will protect the
+identity of the reporter, and treat the content of complaints as confidential
+(unless the reporter agrees otherwise).
+
+In case of severe and obvious breaches, e.g., personal threat or violent, sexist
+or racist language, we will immediately disconnect the originator from pygfx
+communication channels; please see the manual for details.
+
+In cases not involving clear severe and obvious breaches of this code of
+conduct, the process for acting on any received code of conduct violation
+report will be:
+
+1. acknowledgement that the report has been received
+2. reasonable discussion/feedback
+3. mediation (if feedback didn't help, and only if both reporter and reportee agree to this)
+4. enforcement via transparent decision
+
+The committee will respond to any report as soon as possible, and at most
+within 72 hours.
```

Comment:
I think we cannot honor this constraint (within 72 hours). Realistically we are just two persons with limited amounts of time available for this project. I guess we should just stretch it a little, e.g. to a month.